src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,59 @@
+.calendar {
+  display: flex;
+  width: (7 * $day-size) + ($calendar-padding * 2) + (6 * $calendar-gap);
+  box-sizing: border-box;
+  gap: $calendar-gap;
+  padding: $calendar-padding;
+  flex-wrap: wrap;
+  margin: 0 auto;
+
+  &__day {
+    background-color: $calendar-cells-bg;
+    height: $day-size;
+    width: $day-size;
+    border: 1px solid;
+    box-sizing: border-box;
```

Comment:
- don't need to write here box-sizing: border-box, because you did it already in `* {}`
- `* {}` is mean that it will be applied to every tag